_assets/stylesheets/pages/_articles.scss

```diff
@@ -1,189 +1,114 @@
 .article-tpl {
-  > header {
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: cover;
-    margin-bottom: 30px;
-    position: relative;
-
-    p {
-      color: $cr-gray-light;
-      font-size: 13px;
-
-      > a {
-        color: inherit;
-
-        &:hover {
-          color: darken($cr-gray-light, 10);
-          text-decoration: none;
-      }
-        }
-    }
-
-    .article-bg-img {
-      height: 505px;
-      width: 100vw;
-      object-fit: cover;
-
-      @media screen and (min-width: $screen-sm) {
-        height: 775px;
-      }
-    }
-
-    .content-position {
-      position: absolute;
-      bottom: 22%;
-      left: 50%;
-      transform: translate(-50%, 30%);
-        @media screen and (max-width: $screen-sm) {
-           width: 100vw;
-      }
-    }
-  }
-  > .perspective-header {
-    padding: 100px 0 44px;
+  address {
+    background: none;
+    padding: 0px;
   }
 
-  main {
-    font-size: 1.1875rem;
-    line-height: 1.6;
-
-    h1,
-    h2,
-    h3 {
-      font-family: $base-font-face;
-      font-size: 1.5rem;
-      font-weight: 600;
-      line-height: 1.5;
-      margin-top: 0;
+  .article-detail-media-object {
+    .stamp-container {
+      background-color: $cr-white;
     }
-
-    p {
-      margin-bottom: 2rem;
-
-      + h1,
-      + h2,
-      + h3 {
-        margin-top: 60px;
-      }
-      + ul {
-        margin: -2rem 0 2rem 0;
-     }
+    .media-object {
+      color: $cr-gray;
     }
+  }
 
-    ol > li {
-      margin-top: 24px;
+  .article-social-share {
+    a.on > span.heart-o > svg, a.on > span.heart > svg {
```

Comment:
Changes that need to be made to the component should be done in the crds-components repo